src/app_service/service/community/service.nim

```diff
@@ -624,6 +585,54 @@ QtObject:
     except Exception as e:
       error "Error creating community channel", msg = e.msg, communityId, name, description, procName="createCommunityChannel"
 
+  proc createCommunity*(
+    self: Service,
+    name: string,
+    description: string,
+    access: int,
+    ensOnly: bool,
+    color: string,
+    imageUrl: string,
+    aX: int, aY: int, bX: int, bY: int,
+    historyArchiveSupportEnabled: bool
+  ) =
+    try:
+      var image = singletonInstance.utils.formatImagePath(imageUrl)
+      let response = status_go.createCommunity(
+        name,
+        description,
+        access,
+        ensOnly,
+        color,
+        image,
+        aX, aY, bX, bY,
+        historyArchiveSupportEnabled)
+
+      if response.error != nil:
+        let error = Json.decode($response.error, RpcError)
+        raise newException(RpcException, "Error creating community: " & error.message)
+
+      if response.result != nil and response.result.kind != JNull:
+        var community = response.result["communities"][0].toCommunityDto()
+        let communitySettings = response.result["communitiesSettings"][0].toCommunitySettingsDto()
+
+
+        community.settings = communitySettings
+        # add this to the joinedCommunities list and communitiesSettings
+        self.joinedCommunities[community.id] = community
+
+        self.events.emit(SIGNAL_COMMUNITY_CREATED, CommunityArgs(community: community))
+        self.createCommunityChannel(
+          communityId=community.id,
+          name="General",
```

Comment:
Do we let users do channels with uppercase letters? 🤔

Reply:
yes we do, at least Status-go allows it, not sure about the client, I will double check